templates/common/infra/bicep/core/database/postgresql/flexibleserver.bicep

```diff
@@ -0,0 +1,44 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param sku object
+param storage object
+
+param databaseName string
+param administratorLogin string
+@secure()
+param administratorLoginPassword string
+
+// PostgreSQL version
+@allowed(['11', '12', '13', '14', '15'])
```

Comment:
I'd rather not have to update core to support a new version of postgres, so not sure this should be strict.

Reply:
I've removed that, and also updated the PR with changes needed for the Dapr sample. I'd like to do a follow-up PR for Virtual Network support since that's hairier and I won't have time to get into that this week. I left a few comments about changes made in the recent commit. Please take a look, thanks!